tests/adapters/switches/remote_test.py

```diff
@@ -1176,6 +1176,35 @@ def test_disable_interface(self):
 
         self.switch.set_interface_state("ge-0/0/6", OFF)
 
+    def test_unset_interface_state(self):
+        self.requests_mock.should_receive("delete").once().with_args(
+            url=self.netman_url+'/switches/toto/interfaces/ge-0/0/6/shutdown',
+            headers=self.headers
+        ).and_return(
+            Reply(
+                content='',
+                status_code=204))
+
+        self.switch.unset_interface_state("ge-0/0/6")
+
+    def test_unset_interface_state_raises_if_interface_does_not_exist(self):
```

Comment:
in this class, exception marshalling are globally handled and tested, so this test is not required